packages/run-protocol/src/vaultFactory/addAssetToVault.js

```diff
@@ -87,10 +87,37 @@ export const registerScaledPriceAuthority = async ({
     ]),
   ]);
 
-  // we used 4 decimals for ibcAtom above...
-  // 1_000_000 atomBrand = 10_000 ibcAtomBrand
-  const scaleIn = makeRatio(100n, atomBrand, 1n, ibcAtomBrand);
-  const scaleOut = makeRatio(1n, usdBrand, 1n, runBrand);
+  // We need "unit amounts" of each brand in order to get the ratios right.  You
+  // can ignore decimalPlaces when adding and subtracting a brand with itself,
+  // but not when creating ratios.
+  const getDecimalP = async brand => {
+    const displayInfo = E(brand).getDisplayInfo();
```

Comment:
we're veering more from the design that these are merely display info.

if they are truly display info, then couldn't we get this info from somewhere other than the brand? 

if they aren't merely display info, we should revise the design. Not in this PR but if you agree I can make a ticket to follow up.

Reply:
I agree that it's worth getting wider agreement on whether this justification is sound. A ticket seems reasonable; perhaps we can talk about it this afternoon with Zoe stuff.

Reply:
https://github.com/Agoric/agoric-sdk/issues/5348